admin-dev/themes/default/scss/controllers/_carrier-wizard.scss

```diff
@@ -36,7 +37,7 @@
     float: left;
     width: 100%;
     padding: 0;
-    margin: 0 0 10px;
+    margin: 0 0 var(--#{$cdk}size-10);
```

Comment:
These variables are defined in `rem`. They are mainly (though not exclusively) based on multiples of 4. In the future, I think this will lead to better harmonization of spacing and size if we use these variables instead of arbitrary values like (13px / 19px or others).